src/Components/DynamicFields.res

```diff
@@ -210,7 +217,25 @@ let make = (~paymentType, ~list, ~paymentMethod, ~paymentMethodType, ~setRequire
     }, false)
     setAreRequiredFieldsEmpty(._ => areRequiredFieldsEmpty)
     None
-  }, (fieldsArr, email, fullName, country, billingName, line1, line2))
+  }, (
+    fieldsArr,
+    currency,
+    fullName.value,
+    country,
+    billingName.value,
+    line1.value,
+    (
+      email.isValid,
+      email.value,
```

Comment:
Shouldn't giving just the email obj be enough? Do we need to deconstruct it beforehand? If so why?

Reply:
I was debugging the issue with email, that time I added it. Will remove and test once